api/handlers/app.go

```diff
@@ -563,15 +564,9 @@ func (h *App) getProcess(r *http.Request) (*routing.Response, error) {
 	appGUID := routing.URLParam(r, "guid")
 	processType := routing.URLParam(r, "type")
 
-	app, err := h.appRepo.GetApp(r.Context(), authInfo, appGUID)
```

Comment:
We have those sort of gets when listing related resources (e.g. routes for an app) to ensure the user is allowed to get the app, and if not, mask the forbidden error as not found. We did that on purpose in order not to leak information whether a resource ID exists to unauthorised users. Not sure whether we have to remove that.